FlowCal/mef.py

```diff
@@ -71,6 +76,19 @@ def clustering_gmm(data,
     events. For more information, consult ``scikit-learn``'s documentation.
 
     """
+    # Check that ``scale`` is supported
+    if scale!='linear' and scale!='log':
+        raise ValueError("scale {} not supported".format(scale))
+
+    # Rescale if necessary
+    if scale=='log':
+        # Copy events
+        data = data.copy()
+        # Log scale requires saturating negative and zero-valued events
+        data[data < 1] = 1.
+        # Rescale
+        data = np.log10(data)*256
```

Comment:
Parameters `tol` and `min_covar` required extensive tuning for this function to work. If I change the scaling, I'll have to retune these parameters. That's a lot of effort to change something that already works well.